### PR TITLE
feat(android): Add android-navigation and jetpack compose docs

### PR DESCRIPTION
**Pull request by romtsn**

### CODE REVIEW

File: src/platforms/android/configuration/integrations/navigation.mdx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+title: Navigation
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.java.android.navigation
+description: "Learn more about the Sentry Navigation integration for the Android SDK."
+categories:
+  - mobile
+---
+
+The `sentry-android-navigation` library provides [AndroidX Navigation](https://developer.android.com/jetpack/androidx/releases/navigation) support for Sentry using the [SentryNavigationListener](https://github.com/getsentry/sentry-java/blob/a5f30b43b1dad2634ce020809f3b52e0d564a22a/sentry-android-navigation/src/main/java/io/sentry/android/navigation/SentryNavigationListener.kt). The source can be found [on GitHub](https://github.com/getsentry/sentry-java/tree/main/sentry-android-navigation/src/main/java/io/sentry/android/navigation).
```
Comment (romtsn) [inline src/platforms/android/configuration/integrations/navigation.mdx:11]: TODO: change the link to SentryNavListener after it's merged into main

**Comment by mehdialiakbar31** [comment]
> **The latest updates on your projects**. Learn more about [Vercel for Git ↗︎](https://vercel.link/github-learn-more)
> 
> Name	Status	Preview	Updated
> **sentry-docs**	 Ready ([Inspect](https://vercel.com/sentry/sentry-docs/3CMwiLQH99GMMCuDRS7xGYdRbN5K))	[Visit Preview](https://sentry-docs-git-feat-android-navigation.sentry.dev)	Jun 28, 2022 at 10:07PM (UTC)

File: src/platforms/android/configuration/integrations/navigation.mdx (hunk 1 above)
Comment (romtsn) [inline src/platforms/android/configuration/integrations/navigation.mdx:11]:
```suggestion
The `sentry-android-navigation` library provides [AndroidX Navigation](https://developer.android.com/jetpack/androidx/releases/navigation) support for Sentry using the [SentryNavigationListener](https://github.com/getsentry/sentry-java/blob/524ee49b212c3f2eead20960c3c6825ccdbf8007/sentry-android-navigation/src/main/java/io/sentry/android/navigation/SentryNavigationListener.kt). The source can be found [on GitHub](https://github.com/getsentry/sentry-java/tree/main/sentry-android-navigation/src/main/java/io/sentry/android/navigation).
```

File: src/platforms/android/configuration/integrations/navigation.mdx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+Sentry captures data by adding a `NavController.OnDestinationChangedListener`. To add the Navigation integration, initialize the [Android SDK](/platforms/android/), then add the `sentry-android-navigation` dependency. Using Gradle:
+
+```groovy
+implementation 'io.sentry:sentry-android:{{ packages.version('sentry.java.android', '6.2.0') }}'
+implementation 'io.sentry:sentry-android-navigation:{{ packages.version('sentry.java.android.navigation', '6.2.0') }}'
+```
+
+## Configure
+
+Configuration should happen in the respective lifecyce callbacks, once you retrieve your `NavController` instance.
```
Comment (imatwawana) [inline src/platforms/android/configuration/integrations/navigation.mdx:26]:
```suggestion
Configuration should happen in the respective lifecycle callbacks, once you retrieve your `NavController` instance:
```

**Review by imatwawana — APPROVED** [review]
Just a few wording edits but otherwise LGTM!

File: src/platforms/android/configuration/integrations/navigation.mdx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+supportLevel: production
+sdk: sentry.java.android.navigation
+description: "Learn more about the Sentry Navigation integration for the Android SDK."
+categories:
+  - mobile
+---
+
+The `sentry-android-navigation` library provides [AndroidX Navigation](https://developer.android.com/jetpack/androidx/releases/navigation) support for Sentry using the [SentryNavigationListener](https://github.com/getsentry/sentry-java/blob/524ee49b212c3f2eead20960c3c6825ccdbf8007/sentry-android-navigation/src/main/java/io/sentry/android/navigation/SentryNavigationListener.kt). The source can be found [on GitHub](https://github.com/getsentry/sentry-java/tree/main/sentry-android-navigation/src/main/java/io/sentry/android/navigation).
+
+On this page, we get you up and running with Sentry's Navigation Integration, so that it will automatically add a breadcrumb and start an idle transaction for each navigation event.
```
Comment (marandaneto) [inline src/platforms/android/configuration/integrations/navigation.mdx:13]: Similar review on this line https://github.com/getsentry/sentry-docs/pull/5234/files

**Pull request merged**